tests/unit/src/main/scala/tests/TestingServer.scala

```diff
@@ -257,6 +257,9 @@ final class TestingServer(
     }
   }
 
+  def assertShowTasty(filePath: String): Future[Option[String]] =
```

Comment:
As there is no asserts maybe?
```suggestion
  def showTasty(filePath: String): Future[Option[String]] =
```